scripts/sync_repo_files.sh

```diff
@@ -186,6 +188,10 @@ process_repo() {
     git add .
     git commit -s -m "${commit_msg}"
     if push_branch "${org_repo}"; then
+      if [[ "${pr_link}" != "null" ]]; then
+        echo_yellow "Pull request already opened for branch '${branch}': ${pr_link}"
```

Comment:
I think the deletion of the branch would close the PR so we would need to open it again. Let's see.